jedi/api/environment.py

```diff
@@ -136,7 +136,7 @@ def __init__(self):
         self._start_executable = self.executable = sys.executable
         self.path = sys.prefix
         self.version_info = _VersionInfo(*sys.version_info[:3])
-        self._env_vars = {}
+        self._env_vars = None
 
 
 class SameEnvironment(_SameEnvironmentMixin, Environment):
```

Comment:
You seem to have missed the default on line 382 (`_create_environment`).

Reply:
Thanks for catching that! Correction pushed up.